app/assets/stylesheets/config/variables.scss

```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap');
+
+:root{
+  --font-family-main: 'Assistant', sans-serif;
+  --black-background: #181818;
```

Comment:
in scss the correct way to declare variables is with `$` like this:
- `$black-background: #181818;`